src/ddmd/root/rmem.d

```diff
@@ -43,6 +44,14 @@ version (GC)
         {
             return GC.realloc(p, size);
         }
+        static void error() nothrow
```

Comment:
Just a nit, add a blank line between prior and next function declaration.

Reply:
added newline.